recipes/pyopcode/build.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+BUILD_CONFIG=Release
+
+if [ `uname` == Linux ]; then
+    CC=${PREFIX}/bin/gcc
+    CXX=${PREFIX}/bin/g++
+
+    # FIXME refactor to reuse the python name (e.g. python3.5m)
+    # FIXME detect any kind of suffix (m, or d)
+    include_path=${PREFIX}/include/python${PY_VER}
+    if [ ! -d $include_path ]; then
+      # Control will enter here if $DIRECTORY doesn't exist.
+      include_path=${PREFIX}/include/python${PY_VER}m
+    fi
+
+    PY_LIB="libpython${PY_VER}.so"
+    PYTHON_LIBRARY=${PREFIX}/lib/${PY_LIB}
+    if [ ! -f $PYTHON_LIBRARY ]; then
+        PYTHON_LIBRARY=${PREFIX}/lib/libpython${PY_VER}m.so
+    fi
```

Comment:
Yeah, this is fun. I'm ok with leaving it in here for now. Though I would recommend trying to get this into CMake somehow.

I tried to use CMake's "FindPython*" and got frustrated. So, I wrote [my own]( https://github.com/nanshe-org/rank_filter/blob/94b783fbaf20c853b1bec1b64ecfdbaea5d7fb1a/cmake/modules/FindPYTHON.cmake ). Feel free to take it if it helps. Basically it uses the Python interpreter to find out all of this information. Though I haven't tried it on Windows; so, I don't know if it works there or not.